projects/packages/forms/src/contact-form/class-contact-form-field.php

```diff
@@ -1543,23 +1544,17 @@ class=" ' . $classes . '"
 	 * }
 	 */
 	private function get_form_variation_style_properties( $form_style = 'outlined' ) {
-		$style_attrs     = '';
-		$css_vars        = '';
-		$outline_styles  = $this->get_attribute( 'outlinestyledata' );
-		$outline_classes = $this->get_attribute( 'outlinestyleclasses' );
-		$block_name      = 'jetpack/input';
+		$css_vars             = '';
+		$variation_attributes = $this->get_attribute( 'stylevariationattributes' );
+		$variation_attributes = ! empty( $variation_attributes ) ? json_decode( html_entity_decode( $variation_attributes, ENT_COMPAT ), true ) : array();
+		$variation_classes    = $this->get_attribute( 'stylevariationclasses' );
+		$variation_style      = $this->get_attribute( 'stylevariationstyles' );
+		$block_name           = 'jetpack/input';
 
 		if ( $this->maybe_override_type() === 'radio' || $this->maybe_override_type() === 'checkbox-multiple' ) {
 			$block_name = 'jetpack/options';
 		}
 
-		$outline_styles = ! empty( $outline_styles ) ? json_decode( html_entity_decode( $outline_styles, ENT_COMPAT ), true ) : array();
-		$input_styles   = $this->get_attribute( 'inputstyles' );
-
-		if ( ! empty( $input_styles ) ) {
-			$style_attrs = $input_styles;
-		}
```

Comment:
This is where the bug was. I've removed it in favor of `stylevariationstyles` that are generated during shortcode rendering. (see my next comment for where that happens).